ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/security/security.html

```diff
@@ -63,14 +63,29 @@
               <div class="col-xs-3">
 
                 <label for="tls-cert-path">
-                  Path to cerficates folder
+                  Path to certificates folder
                 </label>
 
               </div>
               <div class="col-xs-4">
 
-                <input id="tls-cert-path" class="form-control" type="text" formControlName="tlsCertPath">
+                <label for="tls-cert-path"
+                       aria-haspopup="true"
+                       role="tooltip"
+                       class="form-control tooltip tooltip-validation tooltip-md tooltip-top-left"
+                       [class.invalid]="form.get('tlsCertPath').invalid && (form.get('tlsCertPath').dirty || form.get('tlsCertPath').touched)">
```

Comment:
Let's start thinking how we can abstract common form patterns. For example we could do something like this in a parent class:

 shouldShowError(controlName: string): boolean {
      const control: any = this.form.get(controlName);
      // Errors should show for invalid controls that are either dirty or touched and
      // are not currently active.
      return control.invalid && !control.active && (control.dirty || control.touched);
   }

Then in the template:
[class.invalid]="shouldShowError('tlsCertPath')"

Reply:
I agree. Should we make such modifications here or in a new PR for #40?

Reply:
Yeah we can do it in another PR for #40